packages/experimental/Button/src/Button.types.ts

```diff
@@ -47,26 +28,22 @@ export interface ButtonTokens extends FontTokens, IBorderTokens, IShadowTokens {
   /**
    * The size of the icon.
    */
-  iconSize?: number | string;
+  iconSize?: number;
 
   /**
    * The weight of the lines used when drawing the icon.
    */
   iconWeight?: number;
 
   /**
-   * Text to show on the Button.
+   * The width of the button.
    */
-  content?: string;
```

Comment:
content and icon seemed weird to have as tokens since they're not influencing styling... but if someone feels otherwise I can add them back.